### ISSUE TITLE
Can't open any ods spreadsheet created in LibreOffice 5.2 

### ISSUE THREAD

**Issue by gdefaveri** [pyexcel/pyexcel-ods#15]
Trying to open a ods spreedsheet created in LibreOffice 5.2, even an empty one (see attachment)
[empty.zip](https://github.com/pyexcel/pyexcel-ods/files/569339/empty.zip)

`variablesSheet = pe.get_book(file_name=self.variablesEntry.get())`

I get this error

```
  File "C:\Users\gibo\AppData\Local\Programs\Python\Python35-32\lib\tkinter\__init__.py", line 3118, in insert
    self.tk.call((self._w, 'insert', index, chars) + args)
  File "C:\Users\gibo\AppData\Local\Programs\Python\Python35-32\lib\site-packages\pyexcel\book.py", line 354, in __str__
    return self.texttable
  File "C:\Users\gibo\AppData\Local\Programs\Python\Python35-32\lib\site-packages\pyexcel\book.py", line 363, in custom_presenter
    self.save_to(memory_source)
  File "C:\Users\gibo\AppData\Local\Programs\Python\Python35-32\lib\site-packages\pyexcel\book.py", line 263, in save_to
    source.write_data(self)
  File "C:\Users\gibo\AppData\Local\Programs\Python\Python35-32\lib\site-packages\pyexcel\sources\file_source_output.py", line 97, in write_data
    **self.keywords)
  File "C:\Users\gibo\AppData\Local\Programs\Python\Python35-32\lib\site-packages\pyexcel\sources\rendererfactory.py", line 53, in render_book_to_stream
    self.render_book(book)
  File "C:\Users\gibo\AppData\Local\Programs\Python\Python35-32\lib\site-packages\pyexcel\sources\rendererfactory.py", line 67, in render_book
    self.render_sheet(sheet)
  File "C:\Users\gibo\AppData\Local\Programs\Python\Python35-32\lib\site-packages\pyexcel\sources\renderer\_texttable.py", line 14, in render_sheet
    self.write_title))
  File "C:\Users\gibo\AppData\Local\Programs\Python\Python35-32\lib\site-packages\pyexcel\sources\renderer\_texttable.py", line 25, in render_text_table
    table.set_cols_dtype(['t'] * len(data[0]))
IndexError: list index out of range
```

Same code works as expected with existing ods documents created with previous versions of OpenOffice.

```
pip show pyexcel-ods
---
Metadata-Version: 1.1
Name: pyexcel-ods
Version: 0.2.2
```

**Comment by chfw** [comment]
Reproduced with pyexcel but not with pyexcel-io and pyexcel-ods:

    >>> from pyexcel_io import get_data
    >>> get_data("empty.ods", library="pyexcel-ods")
    OrderedDict([(u'Sheet1', [])])

**Comment by chfw** [comment]
Please try the master branch of pyexcel and see if your issue goes away or not:

    pip install https://github.com/pyexcel/pyexcel/archive/master.zip

**Comment by gdefaveri** [comment]
Yep, that did the trick.

Thanks!

**Comment by chfw** [comment]
That was quick. Will release it within this week.

**Comment by chfw** [comment]
released pyexcel v0.3.3
